### ISSUE TITLE
[Feature Request]: Implement max-width for tearsheet

### ISSUE THREAD

**Issue by kingtraceyj** [carbon-design-system/ibm-products#5897]
### The problem

A PLG team has large imagery in a tear sheet that at large screens, gets too large and pushes content below viewing area. 

![image](https://github.com/user-attachments/assets/7d26f603-e3fb-40d5-9eb0-172c877668c9)

### The solution

To keep images and content from getting too large, we could implement a max-width to keep layouts structured. It was mentioned that composed modal does this so we could follow the same convention with tear sheet. 

### Design link

https://www.figma.com/design/LBqK9ZnkL3TJhryxw4vWW6/What's-new?node-id=0-1&t=HvET5RGN1CWib34r-1

### Owner/main maintainer(s)

Tracey King

### Second/backup maintainer(s)

Bridget Ndege

### Product/offering

PLG - What's new

### Business priority

Medium Priority = upcoming release but is not pressing

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate issues

**Comment by ljcarot** [comment]
@RichKummer  Do you know if this max-width was taken into account in the Tearsheet v2 spec

**Comment by RichKummer** [comment]
@ljcarot Yes! I want to review with the design team, but my thinking is to max out the width of the Tearsheet at `1312px` to display the `xlg` grid.
